frontend/src/i18n/locales/de.yaml

```diff
@@ -125,6 +125,9 @@ video:
   ended: Beendet
   license: Lizenz
   source: Quelle
+  embed:
+    button: embed
```

Comment:
Not sure about the use of lower case here? Yes, it's a verb, but it's also the only thing on that button. It doesn't look right to me :/

Reply:
I was thinking of SO, I think.